src/main/resources/avro/common.avdl

```diff
@@ -2,6 +2,325 @@
 /**
 This protocol defines common types used in the other GA4GH protocols. It does
 not have any methods; it is merely a library of types.
+
```

Comment:
Small nit, but it looks like the indentation on this documentation is off. Should be a 2 space indent.